### ISSUE TITLE
/abandonclaim deletes structures

### ISSUE THREAD

**Issue by rjohnson98126** [GriefPrevention/GriefPrevention#173]
### What behaviour is expected:

expected to not destroy structures or repeatedly crash the client

### What behaviour is observed:

destroyed structures and repeatedly crashed client

### Steps/models to reproduce:

/abandonClaim immediately deleted the entire vertical column of my structure volume without warning, notice, evidence, taunts, lulz, or apology.

random complete client crash occured twice while engaged in normal minecraft noob stuff.   

### Spigot/Craftbukkit/Paper and GriefPrevention version:

CraftBukkit version git-Spigot-99c2a37-9a82fa7 (MC: 1.12) 
API version 1.12-R0.1-SNAPSHOT
GriefPrevention v16.7.1

### GriefPrevention config.yml (if applicable)

none

### Plugin list (if applicable):

no other plugins

**Comment by bigpresh** [comment]
/abandonclaim in a creative rules world will by default remove the contents of the claim - otherwise griefers would claim an area, grief the hell out of it then unclaim it and do the same elsewhere ad nauseum.  That is documented, although the doco is being improved.

With regards to client crashes, I find it very unlikely that that's down to GP - server plugins shouldn't be able to crash the client, and I've certainly not experienced that on my 1.12 server, nor heard any other reports of the same.

**Comment by RoboMWM** [comment]
@rjohnson98126, please paste GP's config.yml and let us know what world this occurred in.

> random complete client crash occured twice while engaged in normal minecraft noob stuff.

This probably means GP wasn't the cause.

**Comment by rjohnson98126** [comment]
I no longer have the info you requested, but i can describe what happened in more detail.   

Background:  I was helping someone else start up a server for a private school.   we wanted to explore grief prevention for obvious reasons and GP seemed like a fit.     i replaced the vanilla MC 1.12 server with Spigot and GP, to a world that had been in place for a few months.   It was a survival world, not creative. 

 the thing i was doing was using Schematica to build a massively huge Space Needle. (~ 20x20x200 ?)   But as /op i built it in creative, because who wants to collect a thousand sea lanterns.   i had claimed the area for building.   after completing the build i abandoned the claim, and immediately a massive column was deleted from the central volume of the structure from bottom to top.   only the blocks i added, but not the entire thing.   just like a shaft right through the center about maybe 5x5x200 (?), but leaving the reset of the structure intact.

well, anyhow, it crashed, i was pissed, then it did it again more or less the same.    i finally fixed it, and then the server owner's teenage son crapped his pants becuase "I cheated".     never mind his 50 mob/loot farms.   whatever.     at this point, i'm the only one in their group who knows what a java command line is, i decided i didn't have the energy to explain what spigot plugins were, and gave up on the whole thing.   they can figure out the whitelist if they have grief kiddies.

TLDR;  i was using beta GP, on an existing world that had been created in vanilla MC, combined with Schematica, and popping in and out of creative.   its probably a miracle the world didnt get random chunks blown to hell everywhere.    so there it is.   thanks for listening.

PS :  GP is really cool.   so despite all my bitchin, i'm looking to use it on a fresh server that I now own.  so to the devs, thanks  :1st_place_medal:

**Comment by RoboMWM** [comment]
Weird. Still would need the info requested to find out what truly happened, but interesting nonetheless. Thanks for getting back to us!
